builtin/credential/ldap/path_config.go

```diff
@@ -367,131 +358,6 @@ func (b *backend) pathConfigWrite(ctx context.Context, req *logical.Request, d *
 	return nil, nil
 }
 
-type ConfigEntry struct {
```

Comment:
This was moved and did bear a minor change - I no longer have a logger on the struct. It felt to me like it would be easy to forget to set the logger, and then possibly get a nil pointer exception, and it wasn't really a config parameter anyways.